lib/src/meta/frappe/doctype.dart

```diff
@@ -337,6 +337,9 @@ class DocType extends FrappeDocument {
       toJson: FrappeDocFieldConverter.boolToCheck)
   bool allowAutoRepeat;
 
+  @JsonKey(ignore: true)
```

Comment:
@nabilshuja I suggest that since it'a a boolean, it's better that it has a defaultValue of `false`.

Reply:
@assemmarwan done